web_app_drawer/static/src/less/variables.less

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+// App Drawer / Icons
+
+@app-drawer-icon-size:      5.625em;
+@app-drawer-icon-margin:    0.625em;
+@app-drawer-width:          80%;
+@app-drawer-title-height:   2.9em;
+
+// Navbar
+
+@navbar-height:  2.3em;
+@navbar-padding-vertical: .565em;
+@navbar-padding-horizontal: .565em;
```

Comment:
Yeah it was option 2 - I'll remove, particularly because it was just there because I thought others would want it 😆